packages/handlers/grpc/src/index.ts

```diff
@@ -7,8 +7,9 @@ import {
   Metadata,
   credentials,
   loadPackageDefinition,
-} from '@grpc/grpc-js';
-import { load } from '@grpc/proto-loader';
+  PackageDefinition,
+} from 'grpc';
+import { createPackageDefinition, load } from '@grpc/proto-loader';
```

Comment:
`createPackageDefinition` isn't exported by `@grpc/proto-loader`.
What can we do to get this?